### ISSUE TITLE
The terminal keeps popping up the error as the terminal process failed to lunch, and its on repeat and cant do anything or can't use any other options  or touch the option.in VS code 

### ISSUE THREAD

**Issue by bonisha89** [microsoft/vscode#128230]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version:
- OS Version:

Steps to Reproduce:

1.
2.

**Comment by bonisha89** [comment]
the bug is because of the update my older version in my other laptop  was working just fine as soon as I updated it it started flickering the whole screen and can't access any of the options or settings. Its just keep popping on the repeat about the terminal process failed.

**Comment by meganrogge** [comment]
what version of vscode are you seeing this on?

**Comment by bonisha89** [comment]
1.58


Thanks,
Bonisha Patel

**Comment by meganrogge** [comment]
any errors in dev tools?

**Comment by bonisha89** [comment]
no, when I run as Administrator it woks just fine. when I open normally the pop up for the terminal error is on the loop.
